guides/writing-addons/intro-tutorial.md

```diff
@@ -45,7 +45,7 @@ Our goal is to be able to pass the `buttonName` value to the addon, just like we
 ```hbs
 <!-- This is a handlebars file in the app using the addon -->
 
-{{addon-name buttonLabel="Register"}}
+<AddonName buttonLabel="Register" />
```

Comment:
```suggestion
<AddonName @buttonLabel="Register" />
```

Reply:
Oops, I remember typing the @.  Don't remember deleting it again.  Getting old I guess.  THanks!